cli-output/src/cli_output.rs

```diff
@@ -303,14 +303,27 @@ pub struct CliValidatorsStakeByVersion {
     pub delinquent_active_stake: u64,
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Copy)]
+pub enum CliValidatorsSortOrder {
```

Comment:
I skipped Version since I didn't want to deal with a semver comparison for now, so lazy..

Reply:
Wfm. Pity we didn't have RpcContactInfo return version as a `semver::Version` -- it implements PartialEq/PartialOrd.